README.old.md

```diff
@@ -0,0 +1 @@
+# react-redux-capstone
```

Comment:
- [ ] it would be better to add tests for details page as the previous reviewer asked in order o complete the requirements

Reply:
This change has been done well